### DISCUSSION TITLE
Debugging: clang/lldb/macOS - no symbols for app but symbols loaded in lldb

### DISCUSSION THREAD

**Discussion by CryptoCrocodile** [microsoft/vscode-cpptools#9867] [Q&A]
Hello,

I'm trying to use vscode to build and debug an app that uses QT. I get symbol information for QT classes, but not for my own classes.
The data in the "variables" panel only shows memory addresses. However, doing a `-exec image lookup -vn MyClassName` shows that there are two locations found where the symbol has been found. Curiously, it's the same location twice. 
Also, the callstack shows proper symbol names as well.
In any case, this indicates that the symbol is available to the lldb debugger, but not in vscode.

Would love to hear about any suggestions. Thanks!

The following is my launch configuration.

> {
    "configurations": [
        {
            "name": "C/C++: Run with Debugger",
            "type": "cppdbg",
            "request": "launch",
            "program": "${workspaceFolder}/${config:buildDirectory}/debug/${config:appName}",
            "args": [],
            "stopAtEntry": false,
            "cwd": "${workspaceFolder}/${config:buildDirectory}/debug",
            "environment": [],
            "externalConsole": false,
            "MIMode": "lldb",
            "symbolLoadInfo": {
                "loadAll": true,
                "exceptionList": "",
            },
            "preLaunchTask": "Build (Debug)",
            "setupCommands": [
                {
                    "text": "settings set target.process.thread.step-avoid-regexp \"\"",
                    "description": "Enable stepping into STL"
                }
            ],
            "logging": {
                "engineLogging": false,
                "trace": false,
                "traceResponse": false
            }
        }
    ],
    "version": "2.0.0"
}

**Comment by sean-mcmanus** [comment]
Can you file an issue (bug)? We can also convert this discussion to an Issue for you if you want.

**Reply by CryptoCrocodile** [reply]
Hi @sean-mcmanus 

Sure, feel free to convert it into an issue. I can provide additional info when needed.

**Reply by CryptoCrocodile** [reply]
Ah, I can create the issue myself. I'll do that.
